src/main/resources/static/common/css/common.css

```diff
@@ -406,3 +406,9 @@ a, a:hover, a:focus {
 .regions {
     max-height: 28px;
 }
+
+.info-icon {
+    font-size: initial;
+    color: #0093d2;
+    vertical-align: text-top;
+}
```

Comment:
It works in both industries Defense and Outdoor

Defense:
![image](https://user-images.githubusercontent.com/109124579/198536039-0b12db30-6bba-40d8-882f-8e8538044b5d.png)

Outdoor:
![image](https://user-images.githubusercontent.com/109124579/198536359-b6c18178-d2d5-4175-80a2-6aed11a4cb70.png)

Reply:
Maybe change this color to be a darker one which is consistent with the "DemoApp OpenAPI" text: #0366AE.

Reply:
Also please put the icon in the middle to be aligned with the text.